contracts/governance/GovernanceSettings.sol

```diff
@@ -7,122 +7,88 @@ import {Ownable} from "@openzeppelin/contracts/access/Ownable.sol";
+    uint256 public taskHandlingReward;
+    /// @notice taskErasingReward is a reward for erasing each task
+    uint256 public taskErasingReward;
+    /// @notice maxOptions is the maximum number of options that can be offered in a single proposal
+    uint256 public maxOptions;
+    /// @notice maxExecutionPeriod is the period after the end of voting during which the proposal can be executed 
+    uint256 public maxExecutionPeriod;
 
     // reverted when proposal fee would be under the sum of burnt fee and rewards
     error ProposalFeeTooLow(uint256 necessaryValue);
 
     constructor() Ownable(msg.sender) {
         // Set default values
-        _proposalBurntFee = 50 * 1e18;
-        _taskHandlingReward = 40 * 1e18;
-        _taskErasingReward = 10 * 1e18;
-        _maxOptions = 10;
-        _maxExecutionPeriod = 72 hours;
-        _proposalFee = _proposalBurntFee + _taskHandlingReward + _taskErasingReward;
-    }
-
-    /// @notice proposalFee is the fee for a proposal
-    /// @return proposal fee
-    function proposalFee() public view returns (uint256) {
-        return _proposalFee;
-    }
-
-    /// @notice proposalBurntFee is the burnt part of fee for a proposal
-    /// @return proposal burn fee
-    function proposalBurntFee() public view returns (uint256) {
-        return _proposalBurntFee;
-    }
-
-    /// @notice taskHandlingReward is a reward for handling each task
-    /// @return task handling reward
-    function taskHandlingReward() public view returns (uint256) {
-        return _taskHandlingReward;
+        proposalBurntFee = 50 * 1e18;
+        taskHandlingReward = 40 * 1e18;
+        taskErasingReward = 10 * 1e18;
+        maxOptions = 10;
+        maxExecutionPeriod = 72 hours;
+        proposalFee = proposalBurntFee + taskHandlingReward + taskErasingReward;
     }
 
-    /// @notice taskErasingReward is a reward for erasing each task
-    /// @return task erasing reward
-    function taskErasingReward() public view returns (uint256) {
-        return _taskErasingReward;
-    }
-
-    /// @notice maxOptions maximum number of options to choose
-    /// @return maximum number of options
-    function maxOptions() public view returns (uint256) {
-        return _maxOptions;
-    }
-
-    /// @notice maxExecutionPeriod is maximum time for which proposal is executable after maximum voting end date
-    /// @return maximum execution period in hours
-    function maxExecutionPeriod() public view returns (uint256) {
-        return _maxExecutionPeriod;
-    }
-
-
     /// @notice setProposalFee sets the fee for a proposal
-    /// @param __proposalFee new proposal fee in native tokens
-    function setProposalFee(uint256 __proposalFee) public onlyOwner {
-        __proposalFee = __proposalFee * 1e18;
-        uint256 necessarySum = _proposalBurntFee + _taskHandlingReward + _taskErasingReward;
-        if(__proposalFee < necessarySum) {
+    /// @param _proposalFee new proposal fee in wei
```

Comment:
it is wei of the native token - you could keep the native tokens in the comment here